core/src/main/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySender.java

```diff
@@ -154,7 +156,6 @@ protected PreparedStatement prepareQueryWithResultSet(Connection con, String que
 			} else {
 				typeNr = JdbcUtil.mapParameterTypeToSqlType(param.getType()).getVendorTypeNumber();
 			}
-			callableStatement.setNull(position, typeNr);
```

Comment:
Should we apply this fix also on 7.8? Lot of projects are still migrating to 7.8, as what I understood. Or is that risky?

Reply:
7.9 only, not 7.8. It does not seem to harm in tests. Only affects those who use stored procedures, which is very few now.

Reply:
Ok, great, thanks!